docs/gallery/interactive/pyfar_plotting.ipynb

```diff
@@ -0,0 +1,400 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plotting and plotstyles\n",
+    "\n",
+    "Plotting and visualizing data is an integral part of data analysis.\n",
+    "pyfar provides functions to plot time signals, frequency spectra, and to create other related data plots.\n",
+    "These can be used to inspect the data or generate scientific plots.\n",
+    "\n",
+    "Under the hood, pyfar uses the matplotlib library for plotting.\n",
+    "This allows for a high degree of customization in the creation of figures.\n",
+    "This notebook provides an overview of the plotting functions and how they can be used to customize the plots for your use-cases.\n",
+    "For more details on the plotting functions, please refer to the [documentation](https://pyfar.readthedocs.io/en/stable/modules/pyfar.plot.html).\n",
+    "\n",
+    "pyfar also adds interactive features to the plots for inspection,",
+    "for example, zooming and cycling through channels.\n",
+    "This can be tested in [this](./pyfar_interactive_plots.ipynb) notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Line plots\n",
+    "\n",
+    "First, we will import pyfar and create a time signal to plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# set matplotlib backend for plotting\n",
+    "%matplotlib inline\n",
+    "\n",
+    "# 1024 samples sine sweep between 100 and 500 Hz\n",
+    "sweep = pf.signals.exponential_sweep_time(2**10, [100, 500])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For a simple time plot, we can use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx-thumbnail": {}
+   },
+   "outputs": [],
+   "source": [
+    "pf.plot.time(sweep)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note, that the plot is styled using the default settings of pyfar.\n",
+    "Checkout the [styling plots section](#styling-plots) for more information on what is changed exactly.\n",
+    "\n",
```

Comment:
```suggestion
```
I think its rather confusing. The reader will see later.

Reply:
Crossrefering to a later chapter here is no benefit and rather confusing

Reply:
This was also suggested by @f-brinkmann so I will leave it for now.